.github/workflows/main.yml

```diff
@@ -25,4 +25,4 @@ jobs:
 
       - run: pnpm prettier --cache --check "**/*.{js,md,ts,tsx}"
 
-      - run: pnpm turbo run size-test build checks
+      - run: pnpm turbo run size-test build checks dts
```

Comment:
small caveat: `checks` include `typecheck`, so essentially now you might be type-checking twice here. Once through `checks` and once through `dts`. I **think** that there is no way to *just* emit declarations without typechecking using `tsc` but I could be wrong here. I think that some other tools might be able to do this since they use TS programmatic API and IIRC there is away to avoid requesting semantic diagnostics there.

Reply:
Sure, I would not fix it now. But some time later.